installer/roles/create-mobile-client-crd/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# tasks file for create-mobile-client-crd
+
+- name: Copy CRD definition to tmp file
+  template:
+    src: ../files/mobile-client-crd.yaml
+    dest: /tmp/mobile-client-crd.yaml  
+
+- name: Create mobileClient CRD
+  shell: "oc create -f /tmp/mobile-client-crd.yaml"
+
+- block:
+  - name: Get admin clusterrole
+    shell: "oc get clusterrole admin -o yaml"
+    register: admin_original_content
+
+  - name: Create new admin clusterrole file
+    template:
+      src: ../files/clusterrole-template.j2
+      dest: /tmp/clusterrole.yaml
+    vars:
+      original_content: "{{ admin_original_content.stdout | from_yaml | to_nice_yaml (indent=2) }}"
+
+  - name: Remove timestamp from admin
```

Comment:
Just curious: why remove the timestamp?

Reply:
The timestamp returned by oc get clusterrole is un-parseable by oc replace. It results in a go error so need to remove it.